CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # Contributing
 
-- Your favourite command isn't covered?
+- Your favorite command isn't covered?
```

Comment:
I'm always wary about doing those changes when editing text. I don't actually have a strong preference either way, but it does seem to me that the American spelling is more prevalent online and the British one always makes me stop and notice it.

Besides, with my spelling checker set to US English I can see words with British spelling underlined in red, but not those with US spelling. So the only way I can make sure documents are consistent is in to harmonize everything to US English (unless there are clearly lots of words using UK spelling).

If that was the only word with a UK/US difference, I'll revert this change in the commit removing the markdown formatting paragraph. Let me know.